### PR TITLE
Example ID added

### PR DESCRIPTION
**Pull request by SartiMau**
TextView added with a character ID example so you can test the app.

Tested in: Samsung Galaxy S8
Signed off by: Mauro Sarti mauro.sarti@globant.com

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,68 +1,75 @@
...
-            android:inputType="number"
-            tools:targetApi="o" />
+            android:layout_height="300dp"
+            android:text="@string/no_character" />
 
-        <LinearLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content">
+    <LinearLayout
+            android:layout_width="250dp"
```
Comment (gastonmira) [inline app/src/main/res/layout/activity_main.xml]: please use dimens for this.

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,68 +1,75 @@
...
 
-            <Button
-                android:id="@+id/buttonSearchRemote"
+        <EditText
+                android:id="@+id/characterID"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_weight="1"
+                android:autofillHints="password"
+                android:drawablePadding="8dp"
```
Comment (gastonmira) [inline app/src/main/res/layout/activity_main.xml]: same here.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,68 +1,75 @@
...
+            android:text="@string/no_character" />
 
-        <LinearLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content">
+    <LinearLayout
+            android:layout_width="250dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginTop="16dp"
```
Comment (gastonmira) [inline app/src/main/res/layout/activity_main.xml]: same here.

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,68 +1,75 @@
...
+            android:id="@+id/textViewDetails"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:autofillHints="password"
-            android:drawablePadding="8dp"
-            android:gravity="center_vertical"
-            android:hint="@string/input_charID"
-            android:inputType="number"
-            tools:targetApi="o" />
+            android:layout_height="300dp"
```
Comment (gastonmira) [inline app/src/main/res/layout/activity_main.xml]: same here.

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (SartiMau) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (SartiMau) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (SartiMau) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (SartiMau) [reply app/src/main/res/layout/activity_main.xml]: done

**Pull request merged**